plugins/alerting/pkg/alerting/api_conditions.go

```diff
@@ -105,3 +110,80 @@ func (p *Plugin) PreviewAlertCondition(ctx context.Context,
 	// return status
 	return nil, shared.AlertingErrNotImplemented
 }
+
+func (p *Plugin) ActivateSilence(ctx context.Context, req *alertingv1alpha.SilenceRequest) (*emptypb.Empty, error) {
```

Comment:
Validate the req parameter here

Reply:
Renamed to `Must` & Panic on failure, since it means the code is written wrong